### ISSUE TITLE
When setting the dayMinWidth using setOption the ability to use scrollToTime doesn't work anymore

### ISSUE THREAD

**Issue by TNAJanssen** [fullcalendar/fullcalendar#7355]
### Reduced Test Case

https://codepen.io/tnajanssen/pen/LYgwLJj

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

React

### Bug Description

I successfully recreated the bug, when you go to the attached codepen and click on scroll to time first it works. When you first click set min width it stops working.

**Comment by acerix** [comment]
Thanks, confirmed.  

The issue seems to be with turning that setting on (which enables horizontal scrolling), it doesn't happen if `dayMinWidth` is set initially so that could work as a workaround:

https://codepen.io/acerix/pen/BaGBGGV

**Comment by TNAJanssen** [comment]
@acerix yes i know, the problem is that i want to change the dayMinWidth according to the number of events so there are not overlapping.

Is there something i can do to help solve this issue as soon as possible? As we are waiting for this to be fixed before we can rollout our new feature.

**Comment by acerix** [comment]
Does it not work to include `dayMinWidth` in the settings like my example? 

I believe this existing ticket captures the need of making events not overlap:
https://github.com/fullcalendar/fullcalendar/issues/3675
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.

**Comment by TNAJanssen** [comment]
No, because it's calculated and with the React version of Fullcalendar I get the same behaviour when the value changes.

I don't know how the React component handles the change of the value underneath, but I can imagine it uses the setoption.

**Comment by TNAJanssen** [comment]
@acerix this is how our view should be, and you're able to scroll to the right:
<img width="2210" alt="image" src="https://github.com/fullcalendar/fullcalendar/assets/2812277/07566b32-b211-4366-a91f-f73a131c02f4">

And the default behaviour is:
<img width="2207" alt="image" src="https://github.com/fullcalendar/fullcalendar/assets/2812277/f04355ac-0da7-49e2-b89c-da52e4e80fbd">

When i set dayMinWidth to a default “big” number like 5000, i get the problem that daily events are centred and you don't know that one exists and the margin between events is too much so you dont have a overview. See the screenshot:
<img width="2210" alt="image" src="https://github.com/fullcalendar/fullcalendar/assets/2812277/908ebda7-c1cd-4b07-a69a-ba9b50ee9e97">

**Comment by acerix** [comment]
It sounds like are using React state to update the value, you can instead use `setOption` through the API which should work as in my example: 

https://fullcalendar.io/docs/react#calendar-api

**Comment by TNAJanssen** [comment]
@acerix yep i did do that, and then this bug entered the game :) That's why i opened this issue.

**Comment by TNAJanssen** [comment]
@acerix when do you think you have more information about this bug?

**Comment by TNAJanssen** [comment]
Any updates?

**Comment by arshaw** [comment]
Hi @TNAJanssen , here's a workaround:
https://codepen.io/arshaw/pen/NWJKJvO?editors=0010

Settings `dayMinWidth: 1` allows it to be set later and maintain scroll. I need to look into it further to determine the root cause.

**Comment by AntoninoBonanno** [comment]
i have same problem. 

`dayMinWidth: 1` not work
